translations/mod-record-specifications/en.json

```diff
@@ -10,7 +12,7 @@
   "Max": "{parameter} must be less than or equal to {value}.",
   "NotNull": "The ''{parameter}'' field is required.",
   "NotBlank": "The ''{parameter}'' must be not blank.",
-  "Size": "The ''{parameter}'' field has exceeded {max} character limit",
+  "Size": "The ''{parameter}'' field is out of allowed bounds [{min}-{max}].",
```

Comment:
The problem with this message is that we will have 0 for min in case only max length is specified in API. It will be not acceptable for required fields like `label`: it couldn't be 0 length.
But we can consult QA.

Reply:
Asked QA, it's ok, clarifying with PO. Otherwise we can just probably add low bound limit as 1

Reply:
Message reverted